src/client/src/ui/spotTile/components/TileHeader.tsx

```diff
@@ -17,12 +18,12 @@ export const CurrencyChartButton = styled('button')`
   border-left: 1px solid white;
 `
 
-const TileHeader: React.SFC<Props> = ({ baseTerm, date, displayCurrencyChart }) => {
+const TileHeader: React.SFC<Props> = ({ baseTerm, date, displayCurrencyChart, publishToChannel }) => {
   const platform = usePlatform()
 
   return (
     <Header>
-      <TileSymbol>{baseTerm}</TileSymbol>
+      <TileSymbol onClick={() => publishToChannel(baseTerm)} style={{ cursor: 'pointer' }}>{baseTerm}</TileSymbol>
```

Comment:
Inline lambdas are not great for performance especially in this part of the application. The method should also be named something more generic like onSymbolClicked.